### PR TITLE
Added OncoKB authentication document

### PR DESCRIPTION
**Pull request by onursumer**
Related to #7053

# Checks
- [ ] Runs on heroku
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

### CODE REVIEW

File: docs/OncoKB-Data-Access.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+A data access token is required to use OncoKB. Please review OncoKB [terms](https://www.oncokb.org/terms)
```
Comment (inodb) [inline docs/OncoKB-Data-Access.md]: maybe good to specify from what cBioPortal version this will be a requirement. We could rephrase it to "a data access token is required **soon**. Then we can merge this to master. The master branch gets automatically deployed to docs.cbioportal.org

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -32,6 +32,7 @@
 * [Authenticating Users via LDAP](Authenticating-Users-via-LDAP.md)
 * [Authenticating and Authorizing Users via Keycloak](Authenticating-and-Authorizing-Users-via-keycloak.md)
 * [Authenticating Users via Tokens](Authenticating-Users-via-Tokens.md)
+* [OncoKB Data Access](OncoKB-Data-Access.md)
```
Comment (inodb) [inline docs/SUMMARY.md]: I'm not sure if auth is the best place to put this. It's more like an integration and has little to do with cBioPortal auth. Maybe we should add it to `2.3 Customization`. That's where we for instance mention CIVIC and others: https://docs.cbioportal.org/2.3-customization/portal.properties-reference. 

It does feel slightly different from customization in that it's more `integration with other web services`. Could also re-organize customization a bit and create new section `2.4 Integration with other web services`. Matchminer and CIVIC could maybe go there as well

File: docs/docker/README.md [hunk 3]
```diff
@@ -53,8 +53,8 @@ above to folders or files that do not exist.
 Copy the
 [`portal.properties.EXAMPLE`](../../src/main/resources/portal.properties.EXAMPLE)
 and change it according to your wishes. See the [full
-reference](portal.properties-Reference.md) and the [skin
-properties](Customizing-your-instance-of-cBioPortal.md) for more information on
```
Comment (onursumer) [inline docs/docker/README.md]: seems like these references has been broken for  a while

File: docs/OncoKB-Data-Access.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+A data access token will be required soon to use OncoKB. Please review OncoKB [terms](https://www.oncokb.org/terms)
```
Comment (zhx828) [inline docs/OncoKB-Data-Access.md]: Should we remove `soon`?

Comment (inodb) [reply docs/OncoKB-Data-Access.md]: just mentioned this to Onur on slack. Let's put the version here. Since this will go in the next release, let's maybe put `from v3.2.3` here

Comment (zhx828) [reply docs/OncoKB-Data-Access.md]: @inodb should be `from v3.2.4` since we already released 3.2.3

**Comment by inodb** [comment]
@onursumer minor thing. I prefer to use the imperative mood for titles of PRs and commits. Makes the titles more concise and easier to read imo: https://chris.beams.io/posts/git-commit/#imperative
